images/build-timing-processor/job-template.yaml

```diff
@@ -18,6 +18,7 @@ spec:
         - name: build-timing-processing-job
           image: ghcr.io/spack/upload-build-timings:0.0.4
           imagePullPolicy: Always
+          command: ["./manage.py", "upload_build_timings"]
```

Comment:
Just to be clear, the plan is to keep the build timings uploader running separately from the django app at first, and to then later fold that into the django webhook handler?

Reply:
Yeah, I already have that change made locally. I just want to verify this PR is deployed and working first before moving more to it.